modules/cnv-configure-liveness-readiness-probes.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc
+
+[id="cnv-configure-liveness-readiness-probes_{context}"]
+This procedure gives you example configuration files you use to define:
+
+* HTTP liveness probes
+* TCP liveness probes
+* Readiness probes
+
+= Define a HTTP liveness probe
+
+.Procedure
+
+* Customize a YAML configuration file to create an HTTP liveness probe, using
+this code block as an example. In this example:
+** You configures a probe using `spec.livenessProbe.httpGet`, which queries port `1500` of the
+`VirtualMachineInstance`, after an initial delay of `120` seconds.
+** The `VirtualMachineInstance` installs and runs a minimal HTTP server
+on port `1500` using `cloud-init`.
++
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+    resources:
+      requests:
+        memory: 1024M
+  livenessProbe:
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    httpGet:
+      port: 1500
+    timeoutSeconds: 10
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    registryDisk:
+      image: registry:5000/kubevirt/fedora-cloud-registry-disk-demo:devel
```

Comment:
this is a very specific URL from our dev environment. Better just use `image: kubevirt/fedora-cloud-registry-disk-demo`.

Reply:
Thanks. Replaced 3 instances of this, one in each section.